### PR TITLE
Sf screen store plugin

### PR DESCRIPTION
**Pull request by lqdung-lockon**
review

### CODE REVIEW

File: src/Eccube/Controller/Admin/Store/OwnerStoreController.php [hunk 1]
```diff
@@ -38,62 +37,71 @@
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpFoundation\Session\Session;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
-/**
- * @Route("/{_admin}/store/plugin/api", service=OwnerStoreController::class)
```
Comment (lqdung-lockon) [inline src/Eccube/Controller/Admin/Store/OwnerStoreController.php]: don't remove route (can remove service)

File: src/Eccube/Controller/Admin/Store/OwnerStoreController.php [hunk 2]
```diff
@@ -38,62 +37,71 @@
...
+    }
+
+
     /**
      * Owner's Store Plugin Installation Screen - Search function
      *
-     * @Route("/search", name="admin_store_plugin_owners_search")
-     * @Template("Store/plugin_search.twig")
-     * @param Application $app
+     * @Route("/%eccube_admin_route%/search", name="admin_store_plugin_owners_search")
```
Comment (lqdung-lockon) [inline src/Eccube/Controller/Admin/Store/OwnerStoreController.php]: remove `%eccube_admin_route%/`

File: src/Eccube/Controller/Admin/Store/PluginController.php [hunk 3]
```diff
@@ -545,7 +556,7 @@ private function getRequestApi(Request $request, $authKey, $url, $app)
         $info['message'] = $message;
         curl_close($curl);
 
-        $app->log('http get_info', $info);
+        $this->logger->info('http get_info', $info);
```
Comment (lqdung-lockon) [inline src/Eccube/Controller/Admin/Store/PluginController.php]: use `log_info` (don't use logger)

File: src/Eccube/Resource/template/admin/Store/plugin_search.twig [hunk 4]
```diff
@@ -44,7 +44,7 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
                     type: 'GET',
                     dataType: 'jsonp',
                     crossDomain: true,
-                    url: '{{ app.config.package_repo_url }}/detail/' + id,
+                    url: '{{ package_repo_url }}/detail/' + id,
```
Comment (lqdung-lockon) [inline src/Eccube/Resource/template/admin/Store/plugin_search.twig]: can use `eccube_config`

**Comment by lqdung-lockon** [comment]
- Fix conflict
- Fix page: install confirm (after click install on the search page)

**Comment by hoand-vn** [comment]
fixed

**Pull request closed without merging**